### DISCUSSION TITLE
How do I split and combine RGBA channels?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [ttddee/Cascade#79] [General]
Hi, how do I split and combine RGBA channels? Was looking to do something like:
![image](https://github.com/ttddee/Cascade/assets/20190653/9fbb12b7-4714-492d-9131-3500e7bc56dc)

**Comment by ttddee** [comment]
Sorry for the late reply.

There is a node called Channel Copy. 

You could do it like this:

![Screenshot from 2023-11-10 07-43-46](https://github.com/ttddee/Cascade/assets/16922381/476a30c1-73e0-4507-a94b-f265474b6cef)
